Tests/TuistDependenciesTests/SwiftPackageManager/SwiftPackageManagerInteractorTests.swift

```diff
@@ -1,28 +1,91 @@
+import TSCBasic
 import TuistCore
+import TuistGraph
 import TuistSupport
 import XCTest
 
 @testable import TuistDependencies
+@testable import TuistDependenciesTesting
 @testable import TuistSupportTesting
 
 final class SwiftPackageManagerInteractorTests: TuistUnitTestCase {
     private var subject: SwiftPackageManagerInteractor!
 
+    private var fileHandlerMock: MockFileHandler!
+
+    private var temporaryDirectoryPath: AbsolutePath!
```

Comment:
You don't need to implement this yourself. `TuistTestCase` already provides a [method](https://github.com/tuist/tuist/blob/main/Sources/TuistSupportTesting/TestCase/TuistTestCase.swift#L85) that lazily creates a temporary directory for you.

```swift
func test_whatever() {
  // Given
  let tmpDir = try self.temporaryPath()

}